src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
It would be better to move this string declaration to a separate file where you can declare your global variables. For example, you could create a file called 'api.ts' and import it from there.